src/clr-icons/all.umd.ts

```diff
@@ -4,9 +4,9 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { ApiModule } from './api';
 import { ClrAllSet } from './shapes/all';
+import { ApiModule } from './api';
 
 /* tslint:disable:variable-name */
-const ClarityIcons = ApiModule.instance;
-ClarityIcons.init(ClrAllSet);
+ApiModule.instance.init();
```

Comment:
Nit: Since `ApiModule.instance.add` calls `init()`, we don't need to initialize, I think.